employee-rostering-backend/pom.xml

```diff
@@ -85,7 +85,7 @@
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-configuration-processor</artifactId>
-      <version>2.1.5.RELEASE</version>
+      <version>2.1.6.RELEASE</version>
```

Comment:
Is there a reason to use different version of `spring-boot-configuration-processor` than the rest of Spring Boot artifacts? Doesn't look like a good idea. All the artifact versions (from one group) should be aligned by using a dependency management from imported BOM or from parent and not specifying the version explicitly.